### ISSUE TITLE
should preserve qouting symbol (single/double) and newline (lf/cr+lf)

### ISSUE THREAD

**Issue by zowers** [jonbretman/amd-to-as6#14]
should preserve qouting symbol (single/double) i.e.
```
define(["underscore"], function(_){})
```
should translate into
```
import _ from "underscore";
```

Also newlines should match original newlines to work fine on windows

thanks!

**Comment by jonbretman** [comment]
Hey, sorry for the late reply. I'll have a look into this.

**Comment by jonbretman** [comment]
Hey, #16 should fix the preserving of quotes and *possibly* the newline issue on windows.

**Comment by jonbretman** [comment]
@zowers If you are able to confirm whether or not the 2 issues you mentioned are fixed as of 0.0.5 that would be awesome. Thanks.

**Comment by zowers** [comment]
yes, works great! thanks
